gpii/configs/mocks/gpii.config.development.local.mock.windows.json5

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
Again, do we need both this and the neighbouring "all.local" variant?

Reply:
Yes. The "local" versions are used by acceptance/integration tests that works together with "gpii.test.pouch.pouchTestCaseHolder" to load test db data from tests/data/dbData and build/tests/dbData. The corresponding "all.local" === its "local" version + pouchManager. The pouchManager loads demo db data from testData/dbData and build/dbData. 

In other words, "local" configs are used for running node tests while "all.local" configs are for starting GPII.

For example, [PCPChannelTests uses `gpii.config.development.local.mock.windows`](https://github.com/cindyli/universal/blob/GPII-2630/gpii/node_modules/flowManager/test/PCPChannelTests.js#L345) while [running PSP channel client example](https://github.com/cindyli/universal/blob/GPII-2630/examples/pcpChannelClient/README.md) requires `gpii.config.development.all.local.mock.windows`.

Reply:
The explanation could be good, but I still don't understand!
When you say "the local" versions work together with pouchTestCaseHolder, but then "all.local" is the same as "local" + pouchManager - isn't this the same thing?
I get that there is a different set of preference sets loaded - that is, the "acceptance test case ones" in tests/dbData and the "snapset ones" in testData/dbData - is there any other difference?
Perhaps could you try writing a little .md file in this directory on the subject "what is the difference between local and all.local"?